### DISCUSSION TITLE
Translations into other languages?

### DISCUSSION THREAD

**Discussion by changh95** [SLAM-Handbook-contributors/slam-handbook-public-release#11] [Ideas]
A big thank you to all the authors for creating such a wonderful book!

This book will easily be the best study resource for anyone looking to learn SLAM seriously. I love how the book covers many of the latest topics too. While the book itself is already a great resource, making this book available in non-English languages will greatly reduce the barrier to learn SLAM for non-English speaking students, so we will have more people with deeper understanding of SLAM globally.

Luckily, with the latest LLM technology, making a first-draft translation of the book is not too difficult (It will take just like an hour to translate the whole book) Eventually, the AI-translation will need to be refined with human inspection to reach publishable quality, but from my experience the first-draft translation is already like 80% good enough and many students can cope with this.

If the authors are happy to get volunteers who will help multi-language translations, we can start working on this. I have a complete OCR text of the book at the moment - I can release it here so people can work on it, given permission from the authors.

![image](https://github.com/user-attachments/assets/54063ebb-d084-4897-bcbb-a4e743ce34d6)

A few steps to do this is:

- Release LaTeX source code of the book (This will make translation progress significantly easier, since we won't need to OCR the entire book. There are also bugs in the PDF as well #10 , so LaTeX source code release will be great)
- AI-translating the book (ChatGPT/Claude/Local LLMs...)
- Compiling the PDF book again
- Human inspection from volunteers

**Comment by ayoungk** [comment]
The book will be published by Cambridge University Press. Any discussion or arrangement regarding translations must be conducted with Cambridge only after the book's completion. At this stage, we are releasing Part 1 of the book solely to address issues and potential bugs. The book is incomplete and subject to significant revisions in future versions. Therefore, any attempt to translate or redistribute the book into other languages at this time is inappropriate and strictly prohibited. Translation-related matters will be formally addressed with Cambridge University Press upon the book’s finalization.

**Reply by changh95** [reply]
Understood! I’ll wait for formal opportunities with Cambridge University Press once the book is finalized.
